back/src/Services/SocketManager.ts

```diff
@@ -809,6 +809,16 @@ export class SocketManager {
         const jwt = Jwt.sign(
             {
                 aud: "jitsi",
+                context: {
+                    user: {
+                        id: user.uuid,
```

Comment:
Can you try changing this with `user.id` ?
If a user enters a Jitsi room several times, we should be there several times. The "uuid" is unique for a user, even if he has several Wokas on the map. The "id" is tied to a Woka so I think we want the id here.